### PR TITLE
Feature sb 29 home page

### PR DESCRIPTION
**Pull request by ValentinaTenus**
1. create home page 
2. add categories carousel with categories and arrow buttons to show more categories or return back.
3. list of categories for now is hardcoded
4. add products filter  by categories
5. add buttons (recommended, justIn and mySize) without functionality
6. add products api and slice
image
image
image

### CODE REVIEW

File: src/pages/home/components/products.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import React, { useCallback, useState } from "react";
+import { useTranslation } from "react-i18next";
+
+import { Product } from "~/libs/types/products.ts";
+
+import { FilterButton } from "./filter-button.tsx";
+import { ProductCard } from "./product-card.tsx";
+import { StyledProductsContainer } from "./styles.ts";
+
+const filters = ["Recommended", "Just In", "Your Size"];
```
Comment (lyudad) [inline src/pages/home/components/products.tsx]: move to translation file

File: src/pages/home/components/styles.ts
Comment (lyudad) [inline src/pages/home/components/styles.ts:1]: not one css to all components, each component should go with own styles file

Comment (lyudad) [reply src/pages/home/components/styles.ts:1]: also why reusable  components are in home folder not ~/components folder?

File: src/pages/home/components/category-item/index.tsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import { Typography } from "@mui/material";
+import React, { useCallback } from "react";
+
+import { colors, fontSizes } from "~/libs/constants/index.ts";
+
+import { type Category } from "../../types/index.ts";
```
Comment (lyudad) [inline src/pages/home/components/category-item/index.tsx]: alias should be used

File: src/pages/home/components/products/index.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+import { FilterButton } from "~/components/index.ts";
+import { Product } from "~/libs/types/products.ts";
+
+import { ProductCard } from "../index.ts";
+import { StyledProductsContainer } from "./styles.ts";
+
+const minNumberOfProducts = 1;
+
+type ProductsProperties = {
+	products: Product[] | undefined;
```
Comment (lyudad) [inline src/pages/home/components/products/index.tsx]: why not used products?: Product[];

File: src/components/loader/index.tsx
Comment (lyudad) [inline src/components/loader/index.tsx:1]: why does the header folder start from a capital letter but the loader no? I assume you dini create rules for the folders, now its mess

File: src/pages/home/components/category-carousel/index.tsx [hunk 4]
```diff
@@ -0,0 +1,70 @@
+import { ArrowBackIos, ArrowForwardIos } from "@mui/icons-material";
+import { Box, IconButton, Typography } from "@mui/material";
+import React from "react";
+import { useTranslation } from "react-i18next";
+
+import { fontSizes } from "~/libs/constants/index.ts";
+
+import { categories } from "../../constants/index.ts";
+import { useCategoryCarousel } from "../../hooks/index.ts";
```
Comment (lyudad) [inline src/pages/home/components/category-carousel/index.tsx]: I ask configure aliases to avoid this

File: src/components/loader/index.tsx
Comment (ValentinaTenus) [reply src/components/loader/index.tsx:1]: we use only lowercase. only several component from capital letter. It wasn't my work, and I wasn't sure I can change it. But I'll do it to make it the same

Comment (ValentinaTenus) [reply src/components/loader/index.tsx:1]: I've tried to change folders and files which were started from a capital letter, but it causes a lot of changes. Can we make separate PR for this?

**Comment by lyudad** [comment]
fonts are wrong

**Comment by ValentinaTenus** [comment]
> fonts are wrong
I've check everything on home page and I've found two wrong fonts( font family for footer links and for price). Were these ones wrong? or there is another issue with fonts?

**Comment by lyudad** [comment]
ignore me, all good you can merge

**Pull request merged**